src/EditorFeatures/Core/GoToDefinition/IGoToDefinitionItemService.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Host;
+
+namespace Microsoft.CodeAnalysis.Editor.GoToDefinition
+{
+    internal interface IGoToDefinitionItemService : ILanguageService
```

Comment:
I would call this more like GoToSymbol as it's not necessarily about Definitions.